### PR TITLE
[bitnami/contour] Sync CRDs with upstream

### PR DESCRIPTION
**Pull request by brenix**
### Description of the change

Updates the CRDs from the upstream project to match the appVersion of the chart.

The update bot has updated the chart for the latest minor release in #16627, but the CRDs were not updated as part of that change.

### Benefits

Ensures the functionality described in the latest release works as expected

### Possible drawbacks

N/A

### Applicable issues

- fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by corico44** [comment]
Hello @brenix,

Could you indicate the sources used from the upstream that you have used to apply these changes, please?

**Comment by brenix** [comment]
Hi @corico44:

The CRD sources were taking from the official project repo here: https://github.com/projectcontour/contour/blob/v1.25.0/examples/contour/01-crds.yaml

File: bitnami/contour/Chart.yaml [hunk 1]
```diff
@@ -22,4 +22,4 @@ maintainers:
 name: contour
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/contour
-version: 12.0.1
+version: 12.0.2
```
Comment (corico44) [inline bitnami/contour/Chart.yaml:25]: @brenix Can you please update your branch with the changes from the main branch? The current branch of contour is 12.0.2 not 12.0.1. So you should also update the version from 12.0.2 to 12.0.3.

Comment (brenix) [reply bitnami/contour/Chart.yaml]: I have rebased and updated the chart version

**Review by corico44 — APPROVED** [review]
LGTM

**Pull request merged**